src/Page/User/Sessions/Sessions.js

```diff
@@ -0,0 +1,120 @@
+import React, { useReducer } from 'react';
+import { Link } from 'react-router-dom';
+import { Card, Table } from 'react-bootstrap';
+import ReactCountryFlag from 'react-country-flag';
+
+import { index } from 'Service/UserSession';
+import timeFormatter from 'Formatter/Time';
+import humanizeFormatter from 'Formatter/Humanize';
+import titleizeFormatter from 'Formatter/Titleize';
+import Pagination from 'Component/Pagination';
+import RequestDetailLocation from 'Component/RequestDetailLocation';
+
+const SET_SESSIONS = 'setSessions';
+const SET_PAGE = 'setPage';
+
+const initialState = {
+  sessions: null,
+  page: 1,
+};
+
+const reducer = (state, { type, payload }) => {
+  switch (type) {
+    case SET_SESSIONS:
+      return {
+        ...state,
+        sessions: payload,
+      };
+
+    case SET_PAGE:
+      return {
+        ...state,
+        sessions: null,
+        page: payload,
+      };
+
+    default:
+      throw new Error(`Invalid action: ${type}`);
+  }
+};
+
+const UserSessions = () => {
+  const [{ sessions, page }, dispatch] = useReducer(reducer, {
+    ...initialState,
+  });
+
+  if (!sessions) {
+    index({ params: { page } }).then(response => {
+      dispatch({ type: SET_SESSIONS, payload: response });
+    });
+  }
+
+  const handlePageSelected = selectedPage => {
+    dispatch({ type: SET_PAGE, payload: selectedPage });
+  };
+
+  return (
+    <Card>
+      <Card.Header>Sessions</Card.Header>
+
+      <Card.Body>
+        <Table responsive>
+          <thead>
+            <tr>
+              <th>Started at</th>
+              <th>Expired At</th>
+              <th>Location</th>
+              <th>Authenticated by</th>
+              <th>Device</th>
+              <th>Role</th>
+              <th>IP</th>
+            </tr>
+          </thead>
+          <tbody>
+            {sessions &&
+              sessions.data.map(session => (
+                <tr key={session.id}>
+                  <td>{timeFormatter(session.created_at)}</td>
+                  <td>{timeFormatter(session.expires_at)}</td>
+                  <td>
+                    {session.request_detail && (
+                      <RequestDetailLocation
+                        requestDetail={session.request_detail}
+                      />
+                    )}
+                    &nbsp;
+                    {session.request_detail &&
+                      session.request_detail.country_code && (
+                        <ReactCountryFlag
+                          code={session.request_detail.country_code}
+                        />
+                      )}
+                  </td>
+                  <td>{session.authenticated_by}</td>
+                  <td>
+                    {session.request_detail &&
+                      session.request_detail.client_name}
+                    on&nbsp;
```

Comment:
```suggestion
                    &nbsp;on&nbsp;
```
![image](https://user-images.githubusercontent.com/7074851/71843973-fa7ea000-3079-11ea-9dfa-25beea626931.png)

Reply:
done!